.github/workflows/test.yml

```diff
@@ -10,6 +10,8 @@ jobs:
       matrix:
         python: [3.5, 3.6, 3.7, 3.8]
     runs-on: ubuntu-latest
+    env:
+      CI: 1
```

Comment:
Why this?

Reply:
@lukicdarkoo made some changes to enables to run the tests locally. Thus here:
https://github.com/cyberbotics/urdf2webots/blob/2d620813c999c38be73f4fbd5d49fdb031ed8b0a/tests/test_export.py#L57
a `CI` ev is used but never defined. The line I added is here to define this ev.